crates/sui-framework/packages/sui-framework/sources/vec_map.move

```diff
@@ -97,6 +97,8 @@ public fun contains<K: copy, V>(self: &VecMap<K, V>, key: &K): bool {
     get_idx_opt(self, key).is_some()
 }
 
+public use fun size as VecMap.length;
+
```

Comment:
Thoughts on creating an extra function here and deprecating the current one?

Reply:
Supportive!